docs/conf.py

```diff
@@ -0,0 +1,197 @@
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+# import os
+# import sys
+# sys.path.insert(0, os.path.abspath('.'))
+
+from __future__ import print_function
+
+import os
+
+import sphinx.environment
+
+# -- General configuration ------------------------------------------------
+
+# If your documentation needs a minimal Sphinx version, state it here.
+#
+# needs_sphinx = '1.0'
+
+# Do not warn on external images.
+suppress_warnings = ['image.nonlocal_uri']
+
+# Add any Sphinx extension module names here, as strings. They can be
+# extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
+# ones.
+extensions = [
+    'sphinx.ext.autodoc',
+    'sphinx.ext.coverage',
+    'sphinx.ext.doctest',
+    'sphinx.ext.graphviz',
+    'sphinx.ext.intersphinx',
+    'sphinx.ext.viewcode',
+]
+
+# Add any paths that contain templates here, relative to this directory.
+templates_path = ['_templates']
+
+# The suffix(es) of source filenames.
+# You can specify multiple suffix as a list of string:
+#
+# source_suffix = ['.rst', '.md']
+source_suffix = '.rst'
+
+# The master toctree document.
+master_doc = 'index'
+
+# General information about the project.
+project = 'reana'
+copyright = '2018, info@reana.io'
+author = 'info@reana.io'
+
+# The version info for the project you're documenting, acts as replacement for
+# |version| and |release|, also used in various other places throughout the
+# built documents.
+#
+# The short X.Y version.
+
+# Get the version string. Cannot be done with import!
+g = {}
+with open(os.path.join('..', 'reana_db', 'version.py'), 'rt') as fp:
+    exec(fp.read(), g)
+    version = g['__version__']
+
+# The full version, including alpha/beta/rc tags.
+release = version
+
+# The language for content autogenerated by Sphinx. Refer to documentation
+# for a list of supported languages.
+#
+# This is also used if you do content translation via gettext catalogs.
+# Usually you set "language" from the command line for these cases.
+language = None
+
+# List of patterns, relative to source directory, that match files and
+# directories to ignore when looking for source files.
+# This patterns also effect to html_static_path and html_extra_path
+exclude_patterns = ['_build', 'Thumbs.db', '.DS_Store']
+
+# The name of the Pygments (syntax highlighting) style to use.
+pygments_style = 'sphinx'
+
+# If true, `todo` and `todoList` produce output, else they produce nothing.
+todo_include_todos = False
+
+
+# -- Options for HTML output ----------------------------------------------
+
+# The theme to use for HTML and HTML Help pages.  See the documentation for
+# a list of builtin themes.
+#
+html_theme = 'alabaster'
+
+# Theme options are theme-specific and customize the look and feel of a theme
+# further.  For a list of options available for each theme, see the
+# documentation.
+#
+html_theme_options = {
+    'logo': 'logo-reana.png',
+    'description': 'REANA-DB',
```

Comment:
Please fill in the two-paragraph description in HTML format, as we usually do, in order to have a nice RTFD display (a) for the platform as a whole and (b) for the component itself.

E.g. check out `reana-cluster` for example:

```
html_theme_options = {
    'logo': 'logo-reana.png',
    'description': """<p>REANA-Cluster is a component of the <a
                      href="http://www.reana.io">REANA</a> reusable and
                      reproducible research data analysis
                      platform.</p><p>REANA-Cluster takes care of deploying and
                      managing cluster services on Kubernetes.</p>""",
    'github_user': 'reanahub',
    'github_repo': 'reana-cluster',
    'github_button': False,
    'github_banner': True,
    'show_powered_by': False,
    'extra_nav_links': {
        'REANA@DockerHub': 'https://hub.docker.com/u/reanahub/',
        'REANA@GitHub': 'https://github.com/reanahub',
        'REANA@Twitter': 'https://twitter.com/reanahub',
        'REANA@Web': 'http://www.reana.io',
    }
}
```